pkg/core/config/tenant.go

```diff
@@ -5,6 +5,7 @@ import (
 	"net/http"
 
 	"github.com/kelseyhightower/envconfig"
+	"github.com/skygeario/skygear-server/pkg/auth/dependency/sso" // tolerant nextimportslint: auth
```

Comment:
😱 😨 😱

Reply:
I think we can avoid importing gear package in this case, although the config parsing may be different.

cc @carmenlau

Reply:
A short discussion with @carmenlau.

We suggest to change the config format like this:

```
SSO_PROVIDERS=google,facebook,skygear

SSO_GOOGLE_CLIENT_ID=my_google_app_id
SSO_GOOGLE_CLIENT_SECRET=my_google_app_secret
SSO_GOOGLE_SCOPE=https://www.googleapis.com/auth/userinfo.email

SSO_FACEBOOK_CLIENT_ID=my_facebook_app_id
SSO_FACEBOOK_CLIENT_SECRET=my_facebook_app_secret
SSO_FACEBOOK_SCOPE=email,user_friends

SSO_SKYGEAR_CLIENT_ID=my_facebook_app_id
SSO_SKYGEAR_CLIENT_SECRET=my_facebook_app_secret
SSO_SKYGEAR_SCOPE=email,user_friends
```

Obviously, skygear is not an auth provider, but the config parsing would not throw error on this, instead the auth package sso struct would throw when it does not recognise the provider name.